packages/core/core/src/types.js

```diff
@@ -45,26 +44,26 @@ export type DependencyNode = {|
 export type FileNode = {|id: string, +type: 'file', value: File|};
 export type RootNode = {|id: string, +type: 'root', value: string | null|};
 
-export type TransformerRequestNode = {|
-  id: string,
-  +type: 'transformer_request',
-  value: TransformerRequest
-|};
+// Asset group nodes are essentially used as placeholders for the results of an asset request
+export type AssetGroup = AssetRequest;
 export type AssetGroupNode = {|
   id: string,
   +type: 'asset_group',
+  // An asset group node is used to
   value: AssetGroup
 |};
 
-export type OldAssetGraphNode =
-  | AssetNode
-  | AssetReferenceNode
-  | DependencyNode
-  | FileNode
-  | RootNode
-  | TransformerRequestNode
-  | BundleGroupNode
-  | BundleReferenceNode;
+export type DepPathRequestNode = {|
```

Comment:
why `DepPathRequestNode` and not just `DependencyRequestNode`? what is the path part? maybe I missed something

Reply:
I thought it helped to be more specific about what we are requesting. Instead of `DependencyRequest` and `DevDepRequest` which makes it unclear how they are different, we would have `DepPathRequest` and `DepVersionRequest` which should be a little more intuitive. It may not be the best name at the moment because right now we are getting more than just the path from the resolvers (`AssetRequestRequest`? 😛 ), but I have some thoughts on that. I think ideally the resolvers shouldn't have to know what an `AssetRequest` is. Their job should only be to return a path. The AssetGraphBuilder can take care of adding the dep's env to the follow up `AssetRequest`. As for the side effects, I was thinking that getting the `package.json` for a file about to be transformed would work well as a `ConfigRequest` once I have that piece implemented. So I changed the name in anticipation of those changes.